.github/workflows/deploy.yml

```diff
@@ -176,3 +176,36 @@ jobs:
       project_type: ui
       azure_app_name: ${{ matrix.azure_app_name }}
     secrets: inherit
+
+  run-api-acceptance-tests:
+    if: inputs.environment == 'Test' || inputs.environment == 'Test2'
+    name: ${{ matrix.job_name }}
+    needs: [ deploy-ui-services ]
```

Comment:
Should we run these in parallel with the UI deployment instead of afterwards? So `needs: [ deploy-api-services ]`

Reply:
I think it is better to run them after everything has deployed and if we add UI tests, they will (hopefully) run in the same pattern.

Reply:
OK changed to startsWith

Reply:
Fair enough, was just thinking if we would want to cancel the UI deployments if acceptance tests on the APIs failed - but since this is only applicable to Test anyway, it makes more sense to just deploy the lot and diagnose